server/tests/events.js

```diff
@@ -0,0 +1,399 @@
+          name: 'test event changed',
+          eventType: 'Conference',
+          date: '12/2/2019',
+          guestNo: 200,
+          email: 'testuser1@gmail.com',
+          centerId: helper.centerId
+        })
+        .end((err, res) => {
+          expect(res).to.be.status(400);
+          res.should.be.a('object');
+          res.body.should.have.property('message').to.equal('There was an error with the event ID input!');
+          done();
+        });
+    });
+
+    it('should return a status 404 response for modifying an event that does not exist', (done) => {
+      chai
+        .request(app)
+        .put(`/api/v1/events/100000?token=${helper.userToken}`)
+        .send({
+          name: 'test event changed',
+          eventType: 'Conference',
+          date: '12/2/2019',
+          guestNo: 200,
+          email: 'testuser1@gmail.com',
+          centerId: helper.centerId
+        })
+        .end((err, res) => {
+          expect(res).to.be.status(404);
+          res.should.be.a('object');
+          res.body.should.have.property('message').to.equal('Event does not exist within our records');
+          done();
+        });
+    });
+
+    // it('should return a status 200 success response for valid put request', (done) => {
+    //   chai
+    //     .request(app)
+    //     .put(`/api/v1/events/${helper.eventId}?token=${helper.userToken}`)
+    //     .send({
+    //       name: 'test event changed',
+    //       eventType: 'Conference',
+    //       date: '12/2/2019',
+    //       guestNo: 200,
+    //       email: 'testuser1@gmail.com',
+    //       centerId: helper.centerId
+    //     })
+    //     .end((err, res) => {
+    //       expect(res).to.be.status(400);
+    //       res.should.be.a('object');
+    //       res.body.should.have.property('message').to.equal('Your event has been updated');
+    //       done();
+    //     });
+    // });
+
+    describe('DELETE api/v1/events/:id', () => {
+      it('should return a status 404 success response for deleting a non existing event', (done) => {
+        chai
+          .request(app)
+          .delete(`/api/v1/events/0?token=${helper.userToken}`)
+          .end((err, res) => {
+            expect(res).to.be.status(404);
+            res.should.be.a('object');
+            res.body.should.have.property('message').to.equal('Event not found');
+            done();
+          });
+      });
+
+      it('should return a status 400 success response for bad input for event Id', (done) => {
+        chai
+          .request(app)
+          .delete(`/api/v1/events/eventID?token=${helper.userToken}`)
+          .end((err, res) => {
+            expect(res).to.be.status(400);
+            res.should.be.a('object');
+            res.body.should.have.property('message').to.equal('There was an error with the event ID input!');
+            done();
+          });
+      });
+
+    //   it('should return a status 200 success response for deleting an event', (done) => {
```

Comment:
I get that you may want to use these commented out  code at some other point in time, but its bad practice to push commented out codes to production.
Please if you are certain you would not be using the code, copy it out and save somewhere else or delete them.

Commented out codes are only pretty in development and not production.
So please fix this.